src/Domain.LinnApps/Stock/StorageLocation.cs

```diff
@@ -114,6 +118,10 @@ public StorageLocation(
 
         public int? StoresKittingPriority { get; set; }
 
+        public string SalesKittableFlag { get; set; }
+
+        public int? SalesKittingPriority { get; set; }
```

Comment:
would make sense to make the setters protected since there are rules like 'sales kittable should be Y, N or blank' to enforce?